tests/integration-tests/tests/schedulers/test_slurm_accounting/resources/require_server_identity.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+CA_URL=$1
+CA_NAME=$2
+CA_PATH="/tmp/slurm"
+CA_FILE_PATH="${CA_PATH}/${CA_NAME}"
+SLURMDBD_CONFIG_FILE="/opt/slurm/etc/slurmdbd.conf"
+
+wget ${CA_URL} -nH -x --cut-dirs=1 -P ${CA_PATH}
```

Comment:
I didn't notice this earlier, but the options `-nH -x --cut-dirs=1` here are useless because they are canceling each other out. Please remove them and keep the command as:
```
wget ${CA_URL} -P ${CA_PATH}
```

Reply:
Good catch. Removed.